src/pages/posts/use-github-actions-for-ci.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Start using GitHub Actions for CI
+date: '2019-03-07'
+spoiler: GitHub Actions can do a lot, including CI. Let's look at how to do it right!
+---
+
+GitHub Actions can do a whole lot of things - [delete branches](https://github.com/jessfraz/branch-cleanup-action), [compose tweets via pull requests](https://github.com/gr2m/twitter-together); it's kind of a general "thing do-er." However, when people hear about Actions, they often ask "is GitHub Actions a replacement for CI providers?" It's a great platform to have CI built into your workflow with minimal setup but a lot of control, so its a totally reasonable question.
+
+I get to use everyones' favorite answer: **it depends**! In this post, I'll share a workflow that I've been using for my Node.js projects, as well as some extensions to it for additional functionality. We'll also take a look at features of popular CI tools and see how they map to GitHub Actions.
+
+## Some pre-reading
+
+I'll be delving into the nitty gritty of writing a workflow file, including some lesser-known functionality. I'd encourage you to read [my previous blog post on GitHub Actions workflows](./what-are-github-workflows), where I talk a bit about workflows as a whole.
+
+You may also want to familiarize yourself with the [actions/bin repo](https://github.com/actions/bin), a collection of actions that are highly scoped and useful for composing a workflow without writing any custom code (especially [actions/bin/filter](https://github.com/actions/bin/blob/master/filter)).
+
+## But I love `{{ ci_provider }}` - why should I care?
+
+You'll see the benefits soon, but I'm not saying that existing projects migrate their CI to actions; rather, new projects benefit from the minimal setup of a CI workflow if you know what you're doing. Are GitHub Actions the _best_ CI platform? I'd say it really depends on your needs; if you're just running test, you can get a lot of functionality with very little effort (I know, the dream :heart_eyes:).
```

Comment:
I imagine you meant to use Actions as the proper noun.

```suggestion
You'll see the benefits soon, but I'm not saying that existing projects migrate their CI to Actions; rather, new projects benefit from the minimal setup of a CI workflow if you know what you're doing. Are GitHub Actions the _best_ CI platform? I'd say it really depends on your needs; if you're just running test, you can get a lot of functionality with very little effort (I know, the dream :heart_eyes:).
```

Reply:
I'm actually torn on this, I need to find a straight answer. When its prefixed with `GitHub` its an uppercase `A`, but without it I'm not sure, I've seen mixed things. Will come back to it l8r sk8r.